packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx

```diff
@@ -325,7 +325,8 @@ const SelectedEntriesModalContent = ({
         }),
       });
 
-      setPublishedCount(res.data.count);
+      // TODO fix for V5
+      setPublishedCount(res.data.data.count);
```

Comment:
is the issue here that data is nested twice?

Reply:
Yes, I've done this for now just to fix the build. This is part of the list view bulk actions which we don't currently have in V5. I think it can be fixed when we work on that